app/screens/send-bitcoin-screen/send-bitcoin-screen.tsx

```diff
@@ -257,11 +293,37 @@ export const SendBitcoinScreen: ScreenType = ({
     setDestinationStatus("NOT_CHECKED")
   }, [destination])
 
-  const pay = useCallback(() => {
+  const fetchInvoice = async (url) => {
+    const response = await fetch(url)
+    return await response.json()
```

Comment:
not sure about the internals of `fetch` but if it throws then the app will crash. We can catch the errors here an return proper `status` and `reason`

Reply:
Good catch (lol), @dolcalmi. I've just pushed up a fix for this.